src/schema/v2/schema.ts

```diff
@@ -134,7 +134,7 @@ const rootFields = {
   partner: Partner,
   // partnerCategories: PartnerCategories,
   // partnerCategory: PartnerCategory,
-  // partners: Partners,
+  partners: PartnersConnection,
```

Comment:
I think we've been preferring the explicit `Connection` camel-cased naming for fields. So `partnersConnection: PartnersConnection`.